Tests/UITests/TestAppUITests/AccountSetupTests.swift

```diff
@@ -105,6 +105,7 @@ final class AccountSetupTests: XCTestCase { // swiftlint:disable:this type_body_
 
     @MainActor
     func testLoginWithMultipleServices() throws {
+        throw XCTSkip("Tests with multiple account services are currently disabled as Spezi is required to support multi same-type modules first.")
```

Comment:
TODO to address this before merging this PR? Or so we merge and then address this in a separate PR once https://github.com/StanfordSpezi/Spezi/pull/110 is merged?

Reply:
Seems to be a bit more complicated to address. https://github.com/StanfordSpezi/Spezi/pull/110 is ready to review and merge. However, this won't address our issue here. We provide both instances as a default value to the `@Dependency` property wrapper. The way it currently works is that we always provide an auto closure to the property wrapper and it will retrieve the first dependency of the same type. So currently, although we provide two different instances, we end up with the same instance twice.

A possible solution is tracked in https://github.com/StanfordSpezi/Spezi/issues/111.

I would propose to merge with the test disabled. We aim to remove multi Account Service support anyways from SpeziAccount. I would suggest to merge this PR as a 2.0.0-beta.1 and iteratively update the design with smaller PRs. This allows us to migrate depending projects step by step.